dynawo/sources/Models/CPP/ModelNetwork/DYNModelShuntCompensator.h

```diff
@@ -358,6 +356,7 @@ class ModelShuntCompensator : public NetworkComponent {
   double ir0_;  ///< initial real part of the current
   double ii0_;  ///< initial imaginary part of the current
   startingPointMode_t startingPointMode_;  ///< type of starting point for the model (FLAT,WARM)
+  bool cannotBeDisconnected_;  ///< true if this shunt cannot be disconnected (due to closed not retained switches in node breaker)
```

Comment:
Not very clear for me at first glance what you intend to do and in which cases you end up here.